client/k8s/application/util/K8sUtil.go

```diff
@@ -437,6 +452,39 @@ func (impl K8sUtil) DeleteJob(namespace string, name string, clusterConfig *Clus
 	return nil
 }
 
+func (impl K8sUtil) GetK8sConfigAndClients(clusterConfig *ClusterConfig) (*rest.Config, *http.Client, *kubernetes.Clientset, error) {
```

Comment:
why client**s**?

Reply:
other clients are used in getting metrics client in some calls.